content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,176 @@ ifdef::backend-html5[]
 :toc: left
 endif::[]
 
-= Administering Jenkins on Kubernetes
\ No newline at end of file
+= Administering Jenkins on Kubernetes
+
+In this section, we will look at:
+
+* Upgrading from one version of Kubernetes to another.
+* How to upgrade the version of Kubernetes in your cluster.
+* Upgrade Kubernetes cluster maintained by the Jenkins Infrastructure project.
+* Upgrade Jenkins Inside a Docker Container.
```

Comment:
Don't upgrade Jenkins inside a container.  Define a new container with the new version, stop the old container, start the new container.
```suggestion
```